tests/console/check_system_info.pm

```diff
@@ -99,8 +99,10 @@ sub run {
 
     script_run('zypper lr | tee /tmp/zypperlr.txt');
 
+    # Need make sure the system is registered then check modules
+    my $output = script_output 'SUSEConnect -s';
     # Check the expected addons before migration
-    if (check_var('VERSION', get_required_var('ORIGIN_SYSTEM_VERSION')) && !get_var("MEDIA_UPGRADE")) {
+    if (check_var('VERSION', get_required_var('ORIGIN_SYSTEM_VERSION')) && ($output !~ /Not Registered/)) {
```

Comment:
It means you won't check offline media and all online cases on basesystem. I think it is better to find a good solution. Maybe add a function in the process of patch_sle before deregister modules.

Reply:
Created a ticket for further investigation. https://progress.opensuse.org/issues/96410